transport/src/main/java/io/netty/channel/pool/ChannelPool.java

```diff
@@ -26,6 +26,25 @@
  * Allows to acquire and release {@link Channel} and so act as a pool of these.
  */
 public interface ChannelPool extends Closeable {
+    /**
+     * Statistics about the {@link ChannelPool}.
+     */
+    interface Stats {
```

Comment:
Is the "stats" terminology already present in Netty? If not I would propose this is renamed to Statistics.